### DISCUSSION TITLE
Brew in fish

### DISCUSSION THREAD

**Discussion by cosmosauce** [Homebrew/discussions#4722] [Getting started]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.5
ORIGIN: https://github.com/Homebrew/brew
HEAD: b8b46501e4caa1cad75b30627c123127ab09a6a7
Last commit: 2 days ago
Core tap JSON: 16 Aug 10:16 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.5.1-arm64
CLT: 14.1.0.0.1.1666437224
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
❯ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  amtk

~ via  v19.9.0 took 16s
```

### Description of issue

I just switched to :tropical_fish:, and I want to install brew on :tropical_fish:; but it only works on the session where I installed it, and if I want to use brew or anything I installed within brew in other sessions, I have to install brew on those sessions, making it very annoying.

I looked through this post,
> fish #4488

but it was also unsolved.

**Comment by ZhongRuoyu** [comment]
It should work after you add the following to `~/.config/fish/fish.config`:

```fish
if status --is-login
  eval (/opt/homebrew/bin/brew shellenv)
end
```

**Comment by SMillerDev** [comment]
The install in fish, same as in bash, tells you how to add the brew path to the system when it finishes.

**Reply by BarbzYHOOL** [reply]
no it doesn't, i mean if you paste the command with forced bash (which is weird)
and i don't know how to run it with fish either
